classes/output/question_attempt_renderer.php

```diff
@@ -127,7 +127,8 @@ public function render_question_attempt(capquiz_question_attempt $attempt, \ques
             ],
             'gradingdone' => $this->capquiz->is_grading_completed(),
             'finalgrade' => $this->capquiz->user()->highest_stars_graded(),
-            'gradingpass' => $this->capquiz->user()->highest_stars_graded() >= $this->capquiz->stars_to_pass()
+            'gradingpass' => $this->capquiz->user()->highest_stars_graded() >= $this->capquiz->stars_to_pass(),
+            'duedate' => $this->capquiz->duedate()
```

Comment:
Use something like `get_string('strftimedate', 'capquiz')` here instead of `duedate()`.
One would have to add the `strftimedate`to the language file: `$string['strftimedate'] = '%d %B %Y';`
```suggestion
            'duedate' => $this->capquiz->get_string('strftimedate', 'capquiz');
```

Reply:
Will make the changes.